entry_types/scrolled/spec/helpers/pageflow_scrolled/entry_json_seed_helper_spec.rb

```diff
@@ -27,8 +28,9 @@ def render(helper, entry)
 
       it 'orders sections according to position attribute' do
         entry = create(:published_entry)
-        section1 = create(:section, revision: entry.revision, position: 4)
-        section2 = create(:section, revision: entry.revision, position: 3)
+        chapter = create(:scrolled_chapter, revision: entry.revision)
+        section1 = create(:section, chapter: chapter, position: 4)
+        section2 = create(:section, chapter: chapter, position: 3)
 
```

Comment:
We also need to make sure now that sections respect the order of their parent chapters.

Reply:
Good catch. I adjusted the code and specs in https://github.com/codevise/pageflow/pull/1288/commits/3dbbf34a19651a4b21c8be2ee507b8d492410620 , following the same pattern as in Pageflow, where the serialization order is maintained by scoping through the entry down to pages. Here I do the same thing but scoping through the main storyline which defines the order of its child models.